pages/shapeshifting-ritual-objects.vue

```diff
@@ -0,0 +1,172 @@
+<template>
```

Comment:
not sure where the code is so just putting the comment here but the select all and clear all is very slow on my machine, with a noticeable delay of a few seconds before the action takes hold. Is it possible to improve this?

Reply:
this is quite instant on my machine but will take a look

Reply:
seem to have got it faster by emitting all at once rather than emitting each filter individually

Reply:
actually had to revert this as it stopped it from filters updating after clear all had been used so unsure how to make this faster, let me know if you have any suggestions

Reply:
I'm going to raise an issue for this and the map markers repopulation as we don't have time to fix this. Will mark this as resolved. Thanks Ruby